rel/server/env.sh.eex

```diff
@@ -12,6 +12,17 @@ if [ "$LIVEBOOK_CLUSTER" = "auto" ] && [ ! -z "$FLY_APP_NAME" ]; then
     deployment="$(echo "$FLY_IMAGE_REF" | shasum | cut -c 1-10)"
     export LIVEBOOK_NODE="${FLY_APP_NAME}-${deployment}@${FLY_PRIVATE_IP}"
   fi
+elsif [ "$LIVEBOOK_CLUSTER" = "auto" ] && [ ! -z "$ECS_CONTAINER_METADATA_URI" ] ; then
+  metadata=`curl --silent ${ECS_CONTAINER_METADATA_URI}`
+  machine_ip=`echo $metadata | $RELEASE_ROOT/bin/livebook eval 'IO.read(:stdio, :eof) |> Jason.decode!() |> Map.fetch!("Networks") |> hd() |> Map.fetch!("IPv4Addresses") |> hd() |> IO.write()'`
+  image_id=`echo $metadata | $RELEASE_ROOT/bin/livebook eval 'IO.read(:stdio, :eof) |> Jason.decode!() |> Map.fetch!("ImageID") |> IO.write()'`
```

Comment:
Just for consistency, since this is the preferred syntax:

```suggestion
  metadata="$(curl --silent $ECS_CONTAINER_METADATA_URI)"
  machine_ip="$(echo $metadata | $RELEASE_ROOT/bin/livebook eval 'IO.read(:stdio, :eof) |> Jason.decode!() |> Map.fetch!("Networks") |> hd() |> Map.fetch!("IPv4Addresses") |> hd() |> IO.write()')"
  image_id="$(echo $metadata | $RELEASE_ROOT/bin/livebook eval 'IO.read(:stdio, :eof) |> Jason.decode!() |> Map.fetch!("ImageID") |> IO.write()')"
```